massgov/mayflower
# Dp 17633 location listing autocomplete

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description
Fixes the geocoding related to autocomplete results on location listings. This mostly rolls back [this commit](https://github.com/massgov/mayflower/pull/745) which was responsible for fixing [this issue](https://jira.mass.gov/browse/DP-15856) in order to produce accurate results with autocomplete, then produces another fix for the above issue to allow partial, non-autocompleted input with properly sorted results.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-17633)

## Steps to Test

1. Go to a location listing page and type in a zip code, city or town name into the search box
2. Select the correct associated result from the autocomplete dropdown
3. Verify the results shown are correct/in the correct order.

To test that there are no regressions for issue [DP-15856](https://jira.mass.gov/browse/DP-15856) - results inaccurate when not using autocomplete - see the associated ticket description.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


*

#### @TODO


*

#### Today I learned...
